_dprhtml/features/bottom-pane/main-pane.html

```diff
@@ -12,7 +12,7 @@
     <div class="main-bottom-pane-tab" title="Conjugation: A tool to conjugate or decline individual words" data-tabid="Cj" data-bind="click: updateActiveTab, css: { 'main-bottom-pane-tab-pressed': isCjTabSelected }"><span>Cj</span></div>
     <div class="main-bottom-pane-tab-separator"></div>
   </div>
-  <div id="main-bottom-pane-tab-panes">
+  <div id="main-bottom-pane-tab-panes" style="overflow: auto">
```

Comment:
hmm... we shouldn't add this here as it impacts all other tabs. Individual tabs decide whether scrollbar is required for them or not. eg. definitions pane does but Cv pane does not.

Reply:
Ok I deleted it but I haven't found another way to make the scrollbars appear in the Cj tab. Adding it to #main-bottom-pane-tab-panes-Cj does not work.

Reply:
Ok scrollbar working now.